### PR TITLE
hides related widget wrapper links

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2016-03-31 at 10 43 04|screen shot 2016-03-31 at 10 51 07|

### CODE REVIEW

File: filer/private/sass/components/_drag-and-drop.scss [hunk 1]
```diff
@@ -2,6 +2,9 @@
 // DRAG AND DROP
 
 form .form-row {
+    .related-widget-wrapper-link {
```
Comment (vxsx) [inline filer/private/sass/components/_drag-and-drop.scss]: this is very unspecific

Comment (vxsx) [inline filer/private/sass/components/_drag-and-drop.scss]: can it affect other fields in the form like this that actually require related-widget stuff?

**Comment by Brankub** [comment]
:+1:

**Comment by vxsx** [comment]
:+1:

**Pull request merged**